.devfile.yaml

```
@@ -0,0 +1,64 @@
+schemaVersion: 2.1.0
+metadata:
+  name: devspaces-demo
+components:
+  - name: tooling-container
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-latest
+      memoryRequest: 256M
+      memoryLimit: 1536M
+      cpuRequest: 0.1
+      cpuLimit: 0.5
```

```suggestion
      cpuRequest: 100m
      cpuLimit: 500m
```
I ran into this issue when starting the workspace on the dogfooding cluster: https://github.com/eclipse/che/issues/21757